ci/run.bat

```diff
@@ -1,4 +1,8 @@
 call dub test --skip-registry=all --compiler=%DC%
+if not %errorlevel% 0 exit /b %errorlevel%
```

Comment:
Shouldn't it be `if %errorlevel% neq 0 exit /b %errorlevel%` ?

Reply:
Maybe that's more accurate.